src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/component/pact/provider/ManageRecallsUIPactTest.kt

```diff
@@ -209,18 +212,19 @@ class ManagerRecallsUiAuthorizedPactTest : ManagerRecallsUiPactTestBase() {
   }
 
   @State(
-    "a recall and uncategorised document exist"
+    "a recall and uncategorised document exist",
+    "a FIXED recall and uncategorised document exist"
```

Comment:
This was what I was mentioning in huddle. I've had to make the actual recalls FIXED because the UI expects the return of the recallLength, and with a standard recall the recallLength is not set & not returned